notebooks/WFC3/calwf3_v1.0_cte/calwf3_with_v1.0_PCTE.ipynb

```diff
@@ -21,23 +21,25 @@
     "- Compare v1.0 and v2.0 products. \n",
```

Comment:
<p>These variable names also can be updated with more descriptive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>